src/promptflow/promptflow/_sdk/_service/entry.py

```diff
@@ -112,36 +111,37 @@ def validate_port(port, force_start):
     if sys.executable.endswith("pfcli.exe"):
         # For msi installer, use sdk api to start pfs since it's not supported to invoke waitress by cli directly
         # after packaged by Pyinstaller.
-        app.logger.info(
-            f"Start Prompt Flow Service on http://localhost:{port}, version: {get_promptflow_sdk_version()}"
-        )
+        app, _ = create_app()
+        if os.environ.get(PF_SERVICE_DEBUG) == "true":
+            app.logger.setLevel(logging.DEBUG)
+        else:
+            app.logger.setLevel(logging.INFO)
+        print(f"Start Prompt Flow Service on http://localhost:{port}, version: {get_promptflow_sdk_version()}")
         waitress.serve(app, host="127.0.0.1", port=port)
     else:
-        # Note: in this scenario, we will have two app, one is the parent process created to log, one is created in
-        # the detached child process.
         # Set host to localhost, only allow request from localhost.
         cmd = ["waitress-serve", f"--listen=127.0.0.1:{port}", "promptflow._sdk._service.entry:get_app"]
         # Start a pfs process using detach mode. It will start a new process and create a new app. So we use environment
         # variable to pass the debug mode, since it will inherit parent process environment variable.
         if platform.system() == "Windows":
+            # For windows, use creationflags=subprocess.CREATE_NEW_PROCESS_GROUP to create a new process group. But
+            # if you close the terminal which start the cmd, the child process will stop since it will send a close
+            # signal to process group started in the terminal.
             subprocess.Popen(cmd, stdout=subprocess.DEVNULL, creationflags=subprocess.CREATE_NEW_PROCESS_GROUP)
         else:
             subprocess.Popen(cmd, stdout=subprocess.DEVNULL, start_new_session=True)
         is_healthy = check_pfs_service_status(port)
         if is_healthy:
-            app.logger.info(
-                f"Start Prompt Flow Service on http://localhost:{port}, version: {get_promptflow_sdk_version()}"
-            )
+            print(f"Start Prompt Flow Service on http://localhost:{port}, version: {get_promptflow_sdk_version()}")
         else:
-            app.logger.warning(f"Pfs service start failed in {port}.")
+            logger.warning(f"Pfs service start failed in {port}.")
 
 
 def stop_service():
-    global app
     port = get_port_from_config()
     if port is not None:
         kill_exist_service(port)
-        app.logger.info(f"Pfs service stop in {port}.")
+        print(f"Pfs service stop in {port}.")
```

Comment:
why print instead of logger here？

Reply:
sdk_cli_logger default logging level is warning and won't show logging level message.
Not sure change to print or just create a new logger?

Reply:
share my opinion: we shall always let the user know PFS related operation (start, stop or restart), so prefer `print`, but need to handle those internal call carefully.

Reply:
But the print may not be logged to the log file，right?